addons/l10n_multilang/models/account.py

```diff
@@ -82,9 +82,3 @@ class AccountAnalyticAccount(models.Model):
     _inherit = 'account.analytic.account'
 
     name = fields.Char(translate=True)
-
-
-class ResCountryState(models.Model):
-    _inherit = 'res.country.state'
-
-    name = fields.Char(translate=True)
```

Comment:
mmh what about https://github.com/odoo/odoo/commit/bc84eb20356e6 ?

I'm not saying I want to keep it, but this was added for a reason :thinking:

Reply:
@fpodoo told me that he discussed with your accounting team and this code can be removed

Reply:
was added by @mart-e , it's not related to accounting. 
It's related to the daily life of people 
* living in a multi-languages country with states
* having the need/wish to translate addresses in different languages (example: Shangai could be translated and printed as 上海 on partner address on invoices/delivery slips...) 

Some found it was a problem when it wasn't translatable, they reported and we agreed at that time. 
Just saying.

Reply:
@qdp-odoo 
I am sure certain people asked to translate the States, because they had a use case. (But it's a minority.)
But I am also sure other peoples had issues on address because states are translated. Translated fields might induce several mistakes.

For instance:
- Suppose I send a letter to: "Virginie-Occidentale", will United States postal services understand that they actually have to send to "West-Virginia"? That's an easy one, but I am sure there are more tricky ones.
- Translations induce a lot of mismatch when people duplicate or edit. (suppose a user edit "New-York" state and rename into "Californie" because they do a mistake; the same customers might receive it's letter in New-York or California, depending on the language)

Reply:
What is the difference with the country (translatable in `base`) and the country state?

> because they had a use case

In which language should you put "Limbourg" in Belgium, "Basel-Stadt" in Switzerland or "eGoli" in South Africa?
Of course, there are more examples, on all continents: https://en.wikipedia.org/wiki/List_of_multilingual_countries_and_regions